src/sass/components/_avatar_editor.scss

```diff
@@ -3,15 +3,16 @@
 
     &.is-dragover {
         .avatar-editor-drag {
-            border-color: black !important;
+            border-color: #000 !important;
```

Comment:
I think I'd add `style: long` (https://github.com/sasstools/sass-lint/blob/master/docs/rules/hex-length.md) to the config for consistency.